### PR TITLE
Baseline maintainers and codeowners

### PR DESCRIPTION
**Pull request by SuZhou-Joe**
### Description
[Describe what this change achieves]

### Issues Resolved
[List any issues this PR will resolve]

### Check List
- [x] Commits are signed per the DCO using --signoff

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

File: MAINTAINERS.md [hunk 1]
```diff
@@ -4,6 +4,10 @@
 | Saurabh Singh          | [getsaurabh02](https://github.com/getsaurabh02)          | Amazon |
 | Subhobrata Dey         | [sbcd90](https://github.com/sbcd90)                      | Amazon |
 | Praveen Sameneni       | [praveensameneni](https://github.com/praveensameneni)    | Amazon |
```
Comment (gaobinlong) [inline MAINTAINERS.md:6]: Seems all maintainers should exist in codeowners file.

File: .github/CODEOWNERS [hunk 2]
```diff
@@ -1,2 +1 @@
-# This should match the owning team set up in https://github.com/orgs/opensearch-project/teams
-*   @opensearch-project/notifications
\ No newline at end of file
+*   @xluo-aws @gaobinlong @Hailong-am @SuZhou-Joe @zhichao-aws @Yuye-Zhu
```
Comment (Hailong-am) [inline .github/CODEOWNERS:1]: yuye-aws?

Comment (gaobinlong) [inline .github/CODEOWNERS:1]: Yuye has changed his Github name, now it's @yuye-aws, and I have to change the name for the backend plugin.

File: MAINTAINERS.md (hunk 1 above)
Comment (SuZhou-Joe) [reply MAINTAINERS.md:6]: Thanks and done

File: .github/CODEOWNERS (hunk 2 above)
Comment (SuZhou-Joe) [reply .github/CODEOWNERS]: Thanks and done.

**Review by xluo-aws — APPROVED** [review]

**Review by Hailong-am — APPROVED** [review]

**Comment by gaobinlong** [comment]
No need to backport 2.x.

File: MAINTAINERS.md [hunk 3]
```diff
@@ -4,6 +4,10 @@
...
+| Xuesong Luo            | [xluo-aws](https://github.com/xluo-aws)       | Amazon      |
+| Hailong Cui            | [Hailong-am](https://github.com/Hailong-am)   | Amazon      |
+| Binlong Gao            | [gaobinlong](https://github.com/gaobinlong)   | Amazon      |
+| Zhou Su                | [SuZhou-Joe](https://github.com/SuZhou-Joe)   | Amazon      |
```
Comment (gaobinlong) [inline MAINTAINERS.md:13]: The format of the MAINTAINERS file should be unified among all repos, see https://github.com/opensearch-project/notifications/blob/main/MAINTAINERS.md, which contains the `Overview` section.

**Comment by SuZhou-Joe** [comment]
@opensearch-project/admin, could anyone help to sync this repo's permissions? Thanks.

File: MAINTAINERS.md (hunk 3 above)
Comment (SuZhou-Joe) [reply MAINTAINERS.md:17]: Thanks and done

**Comment by peternied** [comment]
image
image

**Pull request merged**